synth.py

```diff
@@ -97,6 +97,23 @@
         \g<0>""",
 )
 
+# Make sure that client library version is present in user agent header.
+s.replace(
```

Comment:
Is this something we can update in the Bazel config to avoid manual overrides? https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/BUILD.bazel

Reply:
Not quite familiar with BUILD.bazel, but it looks like pubsub should belong in https://github.com/googleapis/googleapis/tree/master/google/cloud just as pubsublite?